plugins/autochop.cpp

```diff
@@ -197,6 +207,9 @@ static void initialize()
         min_logs = config_autochop.ival(1);
         max_logs = config_autochop.ival(2);
         wait_for_threshold = config_autochop.ival(3);
+        skip_fruit_trees = config_autochop.ival(4);
```

Comment:
Just noticed that you're writing to ival(5-7) but reading from ival(4-6). Anyway, this could probably use just one ival() entry, but it doesn't really matter since (hopefully) we'll use JSON or something else soon.